src/app/pipes/FilterMultipleRecordsPipe.ts

```diff
@@ -6,7 +6,7 @@ export class FilterMultipleRecordsPipe implements PipeTransform {
   transform(multipleRecordsList: any): any {
     if (multipleRecordsList) {
       multipleRecordsList.forEach((techRecordList) => {
-        if (techRecordList) {
+        if (techRecordList && !Array.isArray(techRecordList.techRecord[0])) {
```

Comment:
please rename techRecordList to vehichleTechRecord and simplify the expression too